openfecwebapp/api_caller.py

```diff
@@ -49,14 +49,21 @@ def load_search_results(query, query_type='candidates'):
     return results['results'] if len(results) else []
 
 
-def load_legal_search_results(query, query_type='all', offset=0, limit=20):
+def load_legal_search_results(query, query_type='all', ao_no=None, ao_name=None,
+                                offset=0, limit=20):
     filters = {}
     if query:
         filters['q'] = query
         filters['hits_returned'] = limit
         filters['type'] = query_type
         filters['from_hit'] = offset
 
+        if ao_no and ao_no[0]:
```

Comment:
_sighs_. So, the standard HTML form sends back empty keywords, but we are interpreting them as a list, so we get back results that look like `['']`: a list with a single empty string as an element.